python/lsst/ts/xml/data/sal_interfaces/Script/Script_Events.xml

```diff
@@ -201,5 +201,27 @@
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
+    <item>
+      <EFDB_Name>blockId</EFDB_Name>
+      <Description>Block ID.</Description>
+      <IDL_Type>string</IDL_Type>
+      <IDL_Size>1</IDL_Size>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>dayObs</EFDB_Name>
+      <Description>The dayObs for the block execution, e.g. (20231205).</Description>
```

Comment:
Maybe remove the parenthesis? The type is int and the parenthesis suggest it to be a string. Or include `e.g. ` inside the parenthesis.